python/tvm/auto_scheduler/measure.py

```diff
@@ -740,6 +775,7 @@ def local_run(
     min_repeat_ms=0,
     cooldown_interval=0,
     enable_cpu_cache_flush=False,
+    working_dir="",
```

Comment:
The correctness check is not implemented for LocalRunner.
We should at least add an assertion here.